modules/thrift/CMakeLists.txt

```diff
@@ -39,4 +39,7 @@ zephyr_library_sources_ifdef(CONFIG_THRIFT_SSL_SOCKET
 # needed because std::iterator was deprecated with -std=c++17
 zephyr_library_compile_options(-Wno-deprecated-declarations)
 
+# needed for ctime_r
+zephyr_library_compile_definitions(_POSIX_C_SOURCE=200112L)
```

Comment:
@stephanosio - if I have the opportunity to be picky, we should probably use 200809L here to be consistent with other places in Zephyr.

Reply:
Updated